### PR TITLE
 [MOV] im_livechat: move WebsiteLivechatWindow outside of -

### PR DESCRIPTION
**Pull request by Nemael**
-public_livechat.js

Task-2212347

### CODE REVIEW

File: addons/im_livechat/static/src/legacy/public_livechat.js [hunk 1]
```diff
@@ -3930,4 +3824,3 @@ var DocumentViewer = Widget.extend({
 });
 return DocumentViewer;
 });
-
```
Comment (alexkuhn) [inline addons/im_livechat/static/src/legacy/public_livechat.js:3933]: remove unnecessary diff

**Review by alexkuhn — APPROVED** [review]
@robodoo r+

**Comment by rdeodoo** [comment]
@Nemael You might want to go through all your PRs and check because a lot of them got r+ but actually got stuck.

**Comment by alexkuhn** [comment]
> @Nemael You might want to go through all your PRs and check because a lot of them got r+ but actually got stuck.

No worries, we did not forget to deploy these changes. We actually deployed them soon after in a grouped PR.
All open PRs are either pending and relevant in-progress work that we are actively working on, or already merged changes in a grouped PR and we forgot to close the single PR.

Thanks for letting us know that we should have closed these PRs.

**Comment by rdeodoo** [comment]
> Thanks for letting us know that we should have closed these PRs.

No worries, I am cleaning a bit the rd-website review requests, thought I would mention it :)
